cms/templates/bulksettings.html

```diff
@@ -59,23 +59,36 @@ <h4 class = "section-header-title" title="Collapse/Expand this utility">
                                 <td class = "subsection-header-cell" colspan = "7">
                           <strong><a href="${subsection_settings['url']}">${subsection_settings['name']}</a></strong> -
                           % for setting_type in setting_type_list_map['subsection_setting_types']:
-                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_settings[setting_type]}</span>
+                              <%
+                              subsection_setting = subsection_settings[setting_type] or "N/A"
+                              %>
+                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_setting}</span>
                           % endfor
-                        </td>
+                                </td>
                             </tr>
 
                             % for unit_settings in subsection_settings['children']:
                                 <tr class = "unit-header">
                                     <!-- TODO: dynamically allocate space ?-->
-                                    <td width = "92%" colspan="6"><strong><a href="${unit_settings['url']}">${unit_settings['name']}</a></strong></td>
+                                    <td width = "92%" colspan="6" class="unit-title"><strong><a href="${unit_settings['url']}">${unit_settings['name']}</a></strong></td>
                                     <td width = "8%">${unit_settings['ispublic']}</td> <!--hard coded right now-->
                                 </tr>
 
                                 % for problem_settings in unit_settings['children']:
                                     <tr class="problem-header">
-                                        <td width="52%"><strong><a href="${problem_settings['url']}">${problem_settings['name']}</a></strong></td>
+                                        <%
+                                        problem_name = problem_settings['name'] or "Unnamed Problem"
+                                        %>
+                                        <td width="52%" class="problem-title"><strong><a href="${problem_settings['url']}">${problem_name}</a></strong></td>
```

Comment:
The variable is being called `problem_name` but then the class is called `problem_title`.

Is one more apt than the other?

Should we decide on one (title or name) for consistency?